package-lock.json

```diff
@@ -4,7 +4,6 @@
   "requires": true,
   "packages": {
     "": {
-      "name": "nodejs.org",
```

Comment:
This change doesn't seem related

Reply:
Yes, my bad. Should've noticed the double push